clinicadl/transforms/transforms.py

```diff
@@ -6,9 +6,9 @@
 class Transforms:
     def __init__(
         self,
-        data_augmentation=List[torchio],
-        image_transforms=List[torchio],
-        object_transforms=List[torchio],
+        data_augmentation=List[torchio.Transform],
+        image_transforms=List[torchio.Transform],
+        object_transforms=List[torchio.Transform],
```

Comment:
```suggestion
        data_augmentation=List[Callable],
        image_transforms=List[Callable],
        object_transforms=List[Callable],
```

The user should be able to user other transforms than torchio's.